content/docs/user-guide/pipelines/running-pipelines.md

```diff
@@ -108,6 +108,160 @@ stages:
     always_changed: true
 ```
 
+## Pulling Data
```

Comment:
Since title is a bit general, should we give a one sentence intro- pull gives a way to ... then explain how allow missing helps.

Reply:
Added more in the intro